recipes/dolang/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "dolang" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: de602c48c7b4b6809b8b450cb4a2e3b45408ba372e656f182b93d4508467f2e4
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - numba >=0.49.0
+    - numpy >=1.16
+    - pip
+    - python >=3.7
+    - sympy >=1.5
+    - tempita >=0.5.2
+    - poetry
+  run:
+    - numba >=0.49.0
+    - numpy >=1.16
+    - python >=3.7
+    - sympy >=1.5
+    - tempita >=0.5.2
+
+test:
+  imports:
+    - dolang
+    - dolang.tests
+
+about:
+  home: https://github.com/EconForge/dolang.py
+  license: BSD-2-Clause
+  license_family: BSD
+  license_file: COPYING
```

Comment:
about the `license_file` like, put a comment containing the link the following issue as a reminder to check whether the license file is include in the source distribution for future releases (and so the manually included copy can be removed):

https://github.com/EconForge/dolang.py/issues/3

Reply:
I did that and the issue is solved in upstream dolang repository. Turns out poetry includes COPYING automatically, not LICENSE. In the future, poetry will be able to include either of both but in the meantime I just moved copying to COPYING.